rust/lance-index/src/vector/ivf/shuffler.rs

```diff
@@ -165,8 +165,9 @@ impl IvfShuffler {
             .enumerate();
 
         while let Some((idx, batch)) = stream.next().await {
-            if idx % 100 == 0 {
-                info!("Shuffle Progress {}/{}", idx, total_batch);
+            partition_id = idx / partition_size;
```

Comment:
i think you're missing a `let` here

Reply:
@chebbyChefNEQ what is `partition_size` supposed to be here? It's a `Vec` in the signature. `partition_size[idx]` seems wrong...